tests/utils/test_metric_utils.py

```diff
@@ -0,0 +1,61 @@
+"""Unit tests for metrics utilities.
+
+Author: Travis Driver
+"""
+
+import unittest
+
+import trimesh
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler, Cal3Bundler
+from gtsfm.common.keypoints import Keypoints
+
+import gtsfm.utils.metrics as metric_utils
+
+
+class TestMetricUtils(unittest.TestCase):
+    """Class containing all unit tests for metric utils."""
+
+    def test_mesh_inlier_correspondences(self) -> None:
+        """Tests `compute_keypoint_intersections()` function."""
```

Comment:
same note here as for the function below. it would be nice to have 2-3 sentences explaining the setup so the reader can be extremely lazy and gauge correctness in a single glance at the code after reading the docstring.

Reply:
Comments are very clear already.

Reply:
I like the inline comments. I think a few more sentences would help make it easier for us and others to maintain this code in the future since we're getting into the 100s of unit tests now that we need to maintain.

Reply:
What else needs to be added that's not already in the comments? I'll add it.